src/main/java/io/specto/hoverfly/junit/core/HoverflyConfig.java

```diff
@@ -86,4 +89,44 @@ public boolean isProxyLocalHost() {
         return proxyLocalHost;
     }
 
+    /**
+     * By calling this it means a remote Hoverfly will be used - not started by Java
```

Comment:
should make it clear that the default remote host is localhost, or we always require user to pass host url